depends/config.site.in

```diff
@@ -47,6 +47,10 @@ if test -z $enable_zmq && test -n "@no_zmq@"; then
   enable_zmq=no
 fi
 
+if test -n $disable_bip70 && test -n "@disable_bip70@"; then
+  disable_bip70=no
+fi
```

Comment:
16871: this doesn't quite work I think cause `sed` in `Makefile` looks for `@enable_bip70@`, not   `@disable_bip70@`